terraform/vars/adoagent-stg.tfvars

```diff
@@ -26,10 +26,10 @@ rt_name = "vh-ado-agent-rt-01"
 route_table = {
   route = [
     {
-      name                   = "Route-01"
-      address_prefix         = "10.10.52.0/24"
+      name                   = "ss_stg_vnet"
```

Comment:
this route table needs to have all routes for every environment, as it is an ado agent for every environment